module/os/linux/zfs/zfs_ioctl_os.c

```diff
@@ -264,7 +264,7 @@ _fini(void)
 }
 
 #if defined(_KERNEL)
-module_init(_init);
+module_init(openzfs_init);
 module_exit(_fini);
```

Comment:
it'd probably be best to rename `_fini`, too, for symmetry

Reply:
Sounds reasonable to me. Hopefully the buildbots don't report I somehow ruined everything.